packages/core/types/src/workflow/fulfillment/create-shipping-options.ts

```diff
@@ -1,33 +1,37 @@
-import { ShippingOptionDTO, ShippingOptionPriceType } from "../../fulfillment"
+import { ShippingOptionDTO } from "../../fulfillment"
 import { RuleOperatorType } from "../../common"
 
-export interface CreateShippingOptionsWorkflowInput {
+export type CreateShippingOptionsWorkflowInput = {
```

Comment:
nit(non-blocking): this type is starting to get a bit convoluted – perhaps we can refactor to separate types